### PR TITLE
Documentation including Storybook

### PR DESCRIPTION
**Pull request by johntimothybailey**
# Documentation

See the new CONTRIBUTIONS.md for getting started. This will be the new central / launching area for getting started and even can include FAQs and links to Product information (e.g. the InvisionApp work)

## CLI
To encourage code over written word and rapid feature development for new comers, I am proposing a custom CLI (that leverages [Gluegun](https://infinitered.github.io/gluegun/#/?id=quick-start))

###  Accelerating Project Startup 
Instead of documenting in english all the steps to get start on the app, web, or "functions" (Firebase), do this in code. Doing so also helps the developers

- To quickly spin up the project (right now only focusing on the Native App)
- Having a proving ground for our documentation instead of doing "oh yeah that doesn't actually work"

### Component, Screen, and More generators
Further more by having a CLI we can introduce generators (pull request coming soon!).

`yarn make:screen` and `yarn mark:component`

In doing so we can further help keep consistency and yet drive change by emphasizing changing standards in the generator templates.

### New React Developers Welcome!
By taking an automation or assisted CLI approach we can help new engineers (including Jr. Developers) to be welcome in contributing as they'll have a guide through the CLI prompts and generators.

### TODO
I didn't complete the tests yet :( As there is some scaffolding and mocking you must implement and those that I have from other projects are in TypeScript. I need to either convert them to regular JS or simply make the `kuka/cli` use TypeScript for the tests (which the later would be far easier).

> Totally happy to block this PR without the tests.

## Component Documentation (Product and Development)
By using Storybook we'll encourage Component Driven Development and Design with increased isolation. This gives the added benefit of not having to wait for a page to develop and see components!

To use Storybook see the Storybook section in the CONTRIBUTION.md document.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,11 +0,0 @@
-{
```
Comment (johntimothybailey) [inline package-lock.json:1]: Removing deadcode that caused me initial setup confusion. Thanks @davidrideout for unblocking me!

**Comment by johntimothybailey** [comment]
@davidrideout and @johnkmzhou Happy to rip out Storybook if the rest of this makes sense.

File: CONTRIBUTION.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+```
+
+#### Using Storybook
+Open `.env` and modify `STORYBOOK=true`. The default is false.
+
+Then `$ yarn cache:clear` before running either `$ yarn ios` or `$ yarn android`
+
+
+## Running Usage
+Once the application is running, please signup as there is no "development mode" or "test mode" for Authentication.
```
Comment (johnkmzhou) [inline CONTRIBUTION.md:32]: Most of these instructions can be relegated to the README.md files. CONTRIBUTING.md is more about the process (PRs, issues).

**Review by johnkmzhou — COMMENTED** [review]
I'm not against tests and Storybook (I've used it in the past for the benefit of designers/stakeholders). I personally don't see the need for a cli. It mostly just seems like more code to maintain in a project that at the end of the day is fairly simple and with the goal of getting an MVP. We can discuss this further in the next meeting.

**Pull request closed without merging**